SampleApp/SearchPinsViewController.swift

```diff
@@ -15,7 +15,7 @@ class SearchPinsViewController: MapViewController, UITextFieldDelegate {
 
   override func viewDidLoad() {
     super.viewDidLoad()
-    self.loadScene("scene.yaml", apiKey: "mapzen-2qQR7SX")
+    let _ = try? loadScene("scene.yaml")
     PeliasSearchManager.sharedInstance.urlQueryItems = [NSURLQueryItem(name: "api_key", value: "mapzen-2qQR7SX")]
```

Comment:
Can we update/wrap the search manager to also pull the API key from `MapzenManager`?